srcpkgs/fotoxx/patches/execinfo.patch

```diff
@@ -0,0 +1,17 @@
+diff --git Makefile Makefile
+index 218aa2d..77ec75e 100644
+--- Makefile
++++ Makefile
+@@ -47,4 +47,4 @@ CFLAGS = $(CXXFLAGS) $(CPPFLAGS) -c                \
+    -I/usr/include/libchamplain-0.12/
+
+-LIBS = `$(PKG_CONFIG) --libs gtk+-3.0`                                              \
++LIBS = `$(PKG_CONFIG) --libs gtk+-3.0` $(shell pkg-config --libs libexecinfo)    \
```

Comment:
You should modify `Makefile.am` instead, and then you'll be able to submit it to upstream.

For this you'll need to add `automake` in hostmakedepends and run `autoreconf -fi` at pre_configure().

Reply:
Will try this ASAP

Reply:
hmm ok